client/jetpack-cloud/sections/partner-portal/searchable-dropdown/style.scss

```diff
@@ -0,0 +1,61 @@
+.searchable-dropdown {
+	.components-base-control__field.components-base-control__field {
```

Comment:
You may wonder why this is so. Currently, an inline class is injected into the inner component with styling that always prevails.

Instead of using `!important` , this is another approach to force override.